src/lib/ip-whitelist-backup.json

```diff
@@ -0,0 +1,18 @@
+{
+	"addresses": [
+		"23.235.32.0/20",
```

Comment:
There's an API for getting this https://docs.fastly.com/api/tools#public_ip_list